auth.rst

```diff
@@ -251,6 +251,8 @@ As described in the :ref:`configuration` section, PostgREST accepts a ``jwt-secr
     "n":"9zKNYTaYGfGm1tBMpRT6FxOYrM720GhXdettc02uyakYSEHU2IJz90G_MLlEl4-WWWYoS_QKFupw3s7aPYlaAjamG22rAnvWu-rRkP5sSSkKvud_IgKL4iE6Y2WJx2Bkl1XUFkdZ8wlEUR6O1ft3TS4uA-qKifSZ43CahzAJyUezOH9shI--tirC028lNg767ldEki3WnVr3zokSujC9YJ_9XXjw2hFBfmJUrNb0-wldvxQbFU8RPXip-GQ_JPTrCTZhrzGFeWPvhA6Rqmc3b1PhM9jY7Dur1sjYWYVyXlFNCK3c-6feo5WlRfe1aCWmwZQh6O18eTmLeT4nWYkDzQ"
   }
 
+(this could also be a JSON Web Key Set (JWKS) if it was contained within an array assigned to a `keys` member, e.g. `{ keys: [jwk1, jwk2] }`).
```

Comment:
I think this would fit better in a note, the syntax would be like:

```
.. note::

  This could also be a JSON Web Key Set (JWKS) if it was contained within an array assigned to a ``keys`` member, e.g. ``{ keys: [jwk1, jwk2] }``
```

Also, double backticks are required for the highlighting in reStructuredText.

Reply:
Good idea, I've changed it now.